config/testnet/neofs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+PUBLIC_COLLECTOR="https://collector.fs.neo.org:8428/api/v1/write"
+
+set -m
+neofs-node --config /config.yml &
+vmagent -promscrape.config=/prometheus.yml -remoteWrite.url=${PUBLIC_COLLECTOR}
```

Comment:
vmagent can start before neofs-node pprof server is ready to work. In this case I see in logs
```
error   VictoriaMetrics/lib/promscrape/scrapework.go:258        error when scraping "http://127.0.0.1:9090/metrics" from job "neofs-node-user" with labels {instance="127.0.0.1:9090",job="neofs-node-user",monitor="user_metrics"}: error when scraping "http://127.0.0.1:9090/metrics": dial tcp4 127.0.0.1:9090: connect: connection refused; try -enableTCP6 command-line flag if you scrape ipv6 addresses
```

The best solution is to start vmagent in docker-compose, I guess. But script also can sleep for a while.

Reply:
Agree. I rolled back changes in Dockerfile and added vmagent service into the docker-compose.yml.